awx/urls.py

```diff
@@ -22,6 +23,7 @@
     url(r'^(?:api/)?404.html$', handle_404),
     url(r'^(?:api/)?500.html$', handle_500),
     url(r'^csp-violation/', handle_csp_violation),
+    url(r'^login/', handle_login_redirect),
```

Comment:
cc @matburt because I thought you might have opinions about this - we're adding this as a new endpoint where we can send people who want to do offline login when SAML is enabled

When ui-next is done, we'll update this to point to the proper new URL, but the goal is for _this_ path to be unchanged so we can document it for folks.